styles.css

```diff
@@ -432,3 +440,215 @@ footer .email-footer h3 a {
   font-weight: 400;
   color: #42526e;
 }
+
+/* The block below is for desktop version */
+@media screen and (min-width: 768px) {
+  /* The section below styles the header (logo and menu) */
+  header #navigation {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    height: 60px;
+    background-color: rgba(255, 255, 255, 0);
+    justify-content: center;
+    padding: 0;
+  }
+
+  header #navigation #logo,
+  header #navigation #menu {
+    display: none;
+  }
+
+  header #navigation #nav-list {
+    list-style: none;
+    display: flex;
+    margin-right: 70px;
+    gap: 20px;
+  }
+
+  header #navigation #nav-list li a {
+    text-decoration: none;
+    color: #28352f;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  /* The section below styles the headline section */
+  main .headline-section {
+    background-image: url("img/desktop-bg.png");
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 50px 20%;
+    margin-top: -80px;
+  }
+
+  main .headline-section h1 {
+    font-size: 48px;
+    font-weight: 800;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+  }
+
+  main .headline-section h1 span {
+    font-size: 48px;
+    font-weight: 800;
+    line-height: 60px;
+    color: #36b37e;
+  }
+
+  main .headline-section p {
+    font-size: 20px;
+    font-weight: 500px;
+    line-height: 32px;
+    color: #172b4d;
+  }
+
+  main .headline-section .social-media i {
+    color: #172b4d;
+    font-size: 28px;
+    margin: 15px 8px;
+  }
+
+  /* Scroll to the next section button */
+  main .button-section button {
+    font-size: 30px;
+    color: #6f7780;
+    border-radius: 100%;
+    background-color: #fff;
+    border: 1px solid #6f7780;
+  }
+
+  /* The section below styles the project work section using css grid and flexbox */
+  main .work-section h1 {
+    font-size: 40px;
+  }
+
+  main .work-section #hr-line {
+    display: none;
+  }
+
+  main .work-section .project-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(320px, 1fr));
+    gap: 24px;
+    margin: 0 100px;
+  }
+
+  main .work-section .project-container .work {
+    padding: 0;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    background-color: #ebf0ee;
+    margin: 0;
+  }
+
+  main .work-section .project-container .work .project-info ul {
+    gap: 5px;
+    padding: 8px;
+    margin: 0;
+  }
+
+  main .work-section .project-container .work .project-info ul li {
+    font-size: 12px;
+    padding: 8px;
+  }
+
+  /* The block below styles the about me section and the skills */
+  .about-section {
+    padding: 10px 20%;
+    background-image:
+      url("img/About-bg-2.png"),
+      url("img/about-bg-1.png");
```

Comment:
- [ ] hi you have done really well implementing the desktop version :clap: , I think you have misspelled the names here for the background image files, that is why they are not appearing, please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213201975-6d9a1099-39a1-49ff-acd8-6b9e6d9a165b.png)

I see in your asset file i.e. img/ they are the other way around 
![image](https://user-images.githubusercontent.com/97521557/213202626-1c98c7d8-8e35-47d7-8536-5f0ece5509f3.png)


- [ ] Also please see how in the original design there is some gap between the about section and the works section :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213202300-778ca9db-4a54-4da8-a438-67ea8a2e0e1e.png)

Can we also try to fix this, so our design matches the template as much as possible.